superset-frontend/src/filters/components/Select/SelectFilterPlugin.tsx

```diff
@@ -82,6 +83,7 @@ export default function PluginFilterSelect(props: PluginFilterSelectProps) {
     handleChange(currentValue ?? []);
   }, [
     JSON.stringify(currentValue),
+    sortAscending,
```

Comment:
why not put it in `extraFormData`? and if we not put it here so may be no need to add it this array?

Reply:
Actually this doesn't need to be here, as this won't affect the selected value like the others in the array 👍